Cargo.lock

```diff
@@ -32,6 +82,34 @@ version = "0.2.21"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "683d7910e743518b0e34f1186f92494becacb047c7b6bf616c96772180fef923"
 
+[[package]]
+name = "alloy"
+version = "0.4.0"
```

Comment:
Not a fan that we have yet another alloy version in the dependency tree but oh well